source/plugin/configuration/nodes.rst

```diff
@@ -3,16 +3,24 @@ Configuration Nodes
 ===================
 
 .. javadoc-import::
+    com.google.common.reflect.TypeToken
     ninja.leaping.configurate.ConfigurationNode
+    ninja.leaping.configurate.ConfigurationOptions
     ninja.leaping.configurate.commented.CommentedConfigurationNode
     ninja.leaping.configurate.loader.ConfigurationLoader
+    ninja.leaping.configurate.objectmapping.ObjectMappingException
     ninja.leaping.configurate.objectmapping.serialize.TypeSerializer
+    java.lang.Class
     java.lang.Object
+    java.util.UUID
+    org.spongepowered.api.util.TypeTokens
 
 In memory, the configuration is represented using :javadoc:`ConfigurationNode`\ s. A ``ConfigurationNode`` either holds
 a value (like a number, a string or a list) or has child nodes, a tree-like configuration structure. When using a
 :javadoc:`ConfigurationLoader` to load or create new configurations, it will return the **root node**. It is
-recommended that you always keep a reference to that root node stored somewhere.
+recommended that you always keep a reference to that root node stored somewhere, to prevent loading the configuration
+every time you need to access it and retaining the comments. As an alternative, you could store a reference to a
```

Comment:
"and retaining the comments" is a bit unclear. Can you clarify that a bit?

Reply:
I tried.